code/datums/brain_damage/special_ed.dm

```diff
@@ -0,0 +1,92 @@
+//Brain traumas that are rare and/or somewhat beneficial;
+//they are the easiest to cure, which means that if you want
+//to keep them, you can't cure your other traumas
+/datum/brain_trauma/special
+
+/datum/brain_trauma/special/bluespace_prophet
```

Comment:
Is it possible to get this normally?

Reply:
Should be? Special traumas are a possible receipt when you receive severe traumas.